docs/cmdline-opts/retry-all-errors.d

```diff
@@ -0,0 +1,19 @@
+Long: retry-all-errors
+Help: Retry all errors (use with --retry) (read manpage, don't use by default)
+Added: 7.71.0
+---
+Retry on any error. This option is used together with --retry.
```

Comment:
How about mentioning `--retry-all-errors` in the description of `--retry` to mention the transient errors can be extended to all kinds of errors by `--retry-all-errors`? This will make users notice this new shining option more easily. :)

Reply:
I'd rather not it's a more advanced option.